experimental/framework/tree-react-api/README.md

```diff
@@ -0,0 +1,49 @@
+# @fluid-experimental/tree-react-api
+
+<!-- AUTO-GENERATED-CONTENT:START (README_PACKAGE_SCOPE_NOTICE) -->
+
+**IMPORTANT: This package is experimental.**
+**Its APIs may change without notice.**
+
+**Do not use in production scenarios.**
+
+<!-- AUTO-GENERATED-CONTENT:END -->
+
+Utilities for using SharedTree with React.
+
+This package aims to assist SharedTree based React applications handle some common cases by providing code such applications can share.
+This should improve the quality of such applications by allowing them to share and improve a single implementation of this logic
+(for example ensuring they all handle out of schema documents properly), while reducing the need for boilerplate.
+
+## Known Issues and Limitations
+
+These are a mix of issues encountered when authoring this package, as well as limitation of this package.
+
+Some of this logic would be useful for non-react applications: to avoid creating even more septate packages, that logic is not split into its own package.
+If there is clear demand for this to be done, it might be done in the future.
+
+There is no service implementation agnostic client abstraction that can be referred to here (ex: shared by TinyliciousClient, AzureClient and OdspClient).
+This makes documenting compatibility with that implicit common API difficult.
+It also makes writing service agnostic code at that abstraction level harder.
+
+There does not appear to be a local service implementation of the above mentioned abstraction, which makes testing the code in the package harder.
+
+The commonly used boilerplate for setting up a ContainerSchema based application configures the dev-tools,
+but currently can't be included in this package due to dependency layering issues.
+
+<!-- AUTO-GENERATED-CONTENT:START (README_TRADEMARK_SECTION:includeHeading=TRUE) -->
```

Comment:
There are also section macros for contribution guidelines, general help, etc. that might be worth using. You could also use the mega `LIBRARY_PACKAGE_README` pragma to grab most everything.

Reply:
https://github.com/microsoft/FluidFramework/blob/main/tools/markdown-magic/README.md